example/lib/main.dart

```diff
@@ -22,6 +21,8 @@ class _MyAppState extends State<MyApp> {
       _isDev = false,
       _isExtStorage = false,
       _isEmulator = false;
+  bool _isMockLocationEnabled = false;
+  MethodChannel platform = const MethodChannel('security_plus');
```

Comment:
why using the method channel here while we could use it inside our plugin only

Reply:
"The getter 'isMockLocationEnabled' isn't defined for the type 'SecurityPlus'.
Try importing the library that defines 'isMockLocationEnabled', correcting the name to the name of an existing getter, or defining a getter or field named 'isMockLocationEnabled'." , i got this error when i did that at first and i didn't figure out why its correctly implemented and structured as the other functions

Reply:
Lets check that out tomorrow please